clients/CMakeLists.txt

```diff
@@ -82,6 +82,12 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 
 include( build-options )
 
+if( WIN32 )
+  find_package( hipblas-common REQUIRED CONFIG PATHS "C:/hipSDK")
+else()
+  find_package( hipblas-common REQUIRED CONFIG PATHS /opt/rocm)
+endif()
```

Comment:
Why would the clients find this package?   hipBLAS should capture include paths for its dependencies I think is the way this should go.